plugins/css_loader.js

```diff
@@ -49,13 +50,19 @@ module.exports = new Plugin({
                 document.head.appendChild(window.customCss);
             }
             window.customCss.innerHTML = css;
+            if(window.glasscord)

```

Comment:
This one makes sense if you are having issues with load order.

Reply:
This one will return false 90% of times sadly. I wasn't expecting that.